src/InboundServer/handlers.js

```diff
@@ -307,6 +352,29 @@ const putQuoteById = async (ctx) => {
     ctx.response.status = 200;
 };
 
+/**
+ * Handles a PUT /quotes/{ID}. This is a response to a POST /quotes request
+ */
+const putTransactionRequestsById = async (ctx) => {
+    if(ctx.state.conf.enableTestFeatures) {
+        // we are in test mode so cache the request
+        const req = {
+            headers: ctx.request.headers,
+            data: ctx.request.body
+        };
+        const res = await ctx.state.cache.set(`callback_${ctx.state.path.params.ID}`, req);
+        ctx.state.logger.log(`Cacheing callback: ${util.inspect(res)}`);
+    }
+
+    // publish an event onto the cache for subscribers to action
+    await ctx.state.cache.publish(`tr_${ctx.state.path.params.ID}`, {
```

Comment:
@ksatya77 please use a different channel prefix for transaction requests. "tr_..." is already used by the outbound transfers model; even if there is only small probability of ID clash with transfers I would rather see something different in this model e.g. "trq_...".